File: app/src/main/java/com/example/tanjinc/myapplication/bean/BBean.java
package com.example.tanjinc.myapplication.bean;

import com.example.tanjinc.myapplication.BaseConstant;

public class BBean extends BaseBean {
    String name;

    public BBean(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public int getType() {
        return BaseConstant.TYPE_B;
    }
}

File: app/src/main/java/com/example/tanjinc/myapplication/bean/BBean.kt
package com.example.tanjinc.myapplication.bean

import com.example.tanjinc.myapplication.BaseConstant

class BBean(name: String) : BaseBean() {
    var name: String
        internal set

    init {
        this.name = name
    }

    override fun getType(): Int {
        return BaseConstant.TYPE_B
    }
}
